pkg/grafanaclient/integration_test.go

```diff
@@ -0,0 +1,256 @@
+// +build integration
+
+package grafanaclient
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+var ds = DataSource{Name: "testme",
+	Type:      "influxdb",
+	Access:    "proxy",
+	URL:       "http://localhost:8086",
+	User:      "root",
+	Password:  "root",
+	Database:  "test",
+	IsDefault: true}
+
+var usr = AdminCreateUser{Email: "test@me.com",
+	Login:    "testme",
+	Name:     "testme",
+	Password: "test"}
+
+var org = Org{Name: "testme"}
+
+var org_list = OrgList{ID: 1, Name: "Main Org."}
+
+var org_user = CreateUserInOrg{LoginOrEmail: "testme", Role: "Viewer"}
+
+var url = "http://0.0.0.0:3000"
+var user = "admin"
+var pass = "admin"
+
+func Test_DoLogon(t *testing.T) {
+	session := NewSession(user, pass, url)
+	err := session.DoLogon()
+	assert.Nil(t, err, "We are expecting no error and got one")
```

Comment:
Please log messages like:
  `assert.Nil(t, err, fmt.Sprintf("We are expecting no error and got one err: %s", err))`

It gives much better output like:

```
--- FAIL: Test_DoLogon (0.00s)
        Error Trace:    integration_test.go:38
	Error:      	Expected nil, but got: &url.Error{Op:"Post", URL:"http://127.0.0.1:3000/login", Err:(*net.OpError)(0xc4200e40f0)}
	Messages:   	We are expecting no error and got one err: Post http://127.0.0.1:3000/login: dial tcp 127.0.0.1:3000: getsockopt: connection refused

```